cms/envs/common.py

```diff
@@ -1250,3 +1251,7 @@
 
 # How long until database records about the outcome of a task and its artifacts get deleted?
 USER_TASKS_MAX_AGE = timedelta(days=7)
+
+############## Settings for the Enterprise App ######################
+
+ENTERPRISE_ENROLLMENT_API_URL = LMS_ROOT_URL + "/api/enrollment/v1/"
```

Comment:
Need to define this here, because https://github.com/edx/edx-enterprise/pull/69 now uses `ENTERPRISE_ENROLLMENT_API_URL` as a [class variable for `EnrollmentApiClient`](https://github.com/edx/edx-enterprise/pull/69/files#diff-9bc89aa1bcbe6bccd8de7bbeed2c2f4eR121)

Without this variable defined in CMS envs, the CMS migrations fail to run.